### ISSUE TITLE
To-do list (checklist) not converted correctly

### ISSUE THREAD

**Issue by webmandesign** [akosbalasko/yarle#437]
Hi,

First, thank you for this software!

I'm trying to convert `.enex` files exported from Evernote 10.52.8 and although the conversion with YARLE works, it is not able to convert "to do" lists (lists with checkboxes, checklists).

I've tried to dig deeper and found out that my exported `.enex` file uses this syntax for checklists:
``` html
<ul style="--en-todo:true;">
  <li style="--en-checked:true;">...
```

However, in YARL's `test/data/test-checklist.enex` file this syntax is used instead:
```html
<en-todo checked="false" />
```

If there was an update to how Evernote exports checklists and YARL was not updated to mitigate this change, it may be the cause of this issue.

Currently, all of my checklists are exported as standard lists:
```markdown
* List item 1
* List item 2...
```

Can you please have a look at the issue? Thank you!

**Comment by webmandesign** [comment]
Some more info:
This new Evernote checklist syntax seems to be applied since October 24-27 2022, so possibly Evernote version 10.48?

**Comment by akosbalasko** [comment]
Hi @webmandesign ,

Yes, it sounds that you're right, thank you for reporting it, I'll solve this issue soon.

**Comment by akosbalasko** [comment]
Hi @webmandesign ,

I fixed it in the current release (4.16.0), please take a look at it. I'd close this issue, but do not hesitate to open a new issue, if you still face problems or realize a bug in the app. 
Thanks a lot!

**Comment by webmandesign** [comment]
Working great! Thank you for a fast fix.
